ModelicaReference/package.mo

```diff
@@ -4963,6 +5003,37 @@ discontinuously.]</em></p>
 </html>"));
   end 'smooth()';
 
+  class 'spatialDistribution()' "spatialDistribution()"
+    extends ModelicaReference.Icons.Information;
+    annotation (Documentation(info="<html>
+<p>
+Allows the infinite-dimensional problem below to be solved efficiently with good accuracy.
+</p>
```

Comment:
The actual PDE is missing here.

Reply:
That's why I link to the specification. Really no point to repeat/copy the whole specification text in here now that we can actually link to it directly.

Reply:
But you implicitly refer to it by _below_...

Reply:
Ah right. OK I will remove *below*.